innovation_sweet_spots/pipeline/pilot/investment_predictions/create_dataset.py

```diff
@@ -0,0 +1,167 @@
+"""
+Script to create csv using Crunchbase data that can be
+used for predicting future investment sucess for companies
+"""
+import typer
+from innovation_sweet_spots import PROJECT_DIR
+from innovation_sweet_spots.getters.crunchbase import (
+    get_crunchbase_funding_rounds,
+    get_crunchbase_ipos,
+    get_crunchbase_acquisitions,
+)
+import utils
+import pandas as pd
+
+
+def create_dataset(
+    window_start_date: str = "01/01/2014", window_end_date: str = "01/01/2018"
+):
+    """Loads crunchbase data, processes and saves dataset which can be used
+    to predict future investment success for companies
+
+    Args:
+        window_start_date: Start date for window that simulates the
+            evaluation period for assessing companies. Defaults to "01/01/2014".
+        window_end_date: End date for window that simulates the evaluation period
+            for assessing companies. Defaults to "01/01/2018".
+    """
+    # Date information
+    window_start_date = pd.to_datetime(window_start_date)
+    window_end_date = pd.to_datetime(window_end_date)
+    success_start_date = window_end_date
+    pilot_outputs = PROJECT_DIR / "outputs/finals/pilot_outputs/"
+    # Load datasets
+    cb_orgs = pd.read_csv(pilot_outputs / "ISS_pilot_Crunchbase_companies.csv")
+    cb_acquisitions = get_crunchbase_acquisitions()
+    cb_ipos = get_crunchbase_ipos()
+    cb_funding_rounds = get_crunchbase_funding_rounds()
+
+    # Binarise columns
+    cols_to_binarise = [
+        "twitter_url",
+        "email",
+        "phone",
+        "facebook_url",
+        "homepage_url",
+        "linkedin_url",
+    ]
+    for col in cols_to_binarise:
+        cb_orgs = utils.convert_col_to_has_col(df=cb_orgs, col=col, drop=True)
+
+    # Select relevant cols
+    (
+        cb_orgs[
+            [
+                "id",
+                "name",
+                "legal_name",
+                "description",
+                "location_id",
+                "tech_category",
+                "has_email",
+                "has_phone",
+                "has_facebook_url",
+                "has_twitter_url",
+                "has_homepage_url",
+                "has_linkedin_url",
+                "founded_on",
+                "closed_on",
```

Comment:
Perhaps it's good to define these at the top of of the script (as global 'parameters') to make them more visible?

Although, if the functions you need to call in the chain might change depending on the columns (ie, you can't just simply add or remove a columns without changing the script) then perhaps make sense to keep these here.